R/lime.R

```diff
@@ -46,7 +46,7 @@
 #'
 #' @export
 lime <- function(x, model, ...) {
-  UseMethod('lime')
+  UseMethod("lime")
```

Comment:
Sorry, you are right, I followed the advises of linter...
Do you want me to put single quotes everywhere?